publication-commons/src/main/java/no/unit/nva/publication/service/impl/UpdateResourceService.java

```diff
@@ -118,13 +118,17 @@ public void updateOwner(SortableIdentifier identifier, UserInstance oldOwner, Us
 
     public Resource updateResource(Resource resource, UserInstance userInstance) {
         var persistedResource = fetchExistingResource(resource.toPublication());
-        resource.setCreatedDate(persistedResource.getCreatedDate());
-        resource.setModifiedDate(clockForTimestamps.instant());
 
-        updateCuratingInstitutions(resource, persistedResource);
+        if (resource.hasAffectiveChanges(persistedResource)) {
+            resource.setCreatedDate(persistedResource.getCreatedDate());
```

Comment:
This makes me think that we should have a broader chat about how we (don't) mutate objects